static/scenes/modvil/elements/feed.json5

```diff
@@ -10,62 +10,62 @@
   },
   "cards": [
 /* swapped for mask versions */
-    // "babybird.png",
-    // "badsweater.png",
-    // "bjornpenguin.png",
-    // "chiquitasanta.png",
-    // "closecall.png",
-    // "crossing.png",
-    // "dunks.png",
-    // "elfpower.png",
-    // "fireplacehangout.png",
-    // "grocery.png",
-    // "hanging.png",
-    // "kickflip.png",
-    // "route66.png",
-    // "santaarms.png",
-    // "santacone.png",
-    // "santasled.png",
-    // "snowangels.png",
-    // "snowball.png",
-    // "snowcones.png",
-    // "split.png",
-/* masked versions */
-    "babybird_mask.png",
-    "badsweater_mask.png",
-    "bjornpenguin_mask.png",
-    "chiquitasanta_mask.png",
-    "closecall_mask.png",
-    "crossing_mask.png",
-    "dunks_mask.png",
-    "elfpower_mask.png",
-    "fireplacehangout_mask.png",
-    "grocery_mask.png",
-    "hanging_mask.png",
-    "kickflip_mask.png",
-    "route66_mask.png",
-    "santaarms_mask.png",
-    "santacone_mask.png",
-    "santasled_mask.png",
-    "snowangels_mask.png",
-    "snowball_mask.png",
-    "snowcones_mask.png",
-    "split_mask.png",
-/* new for 2020, mask only */
-    "socialdistance.png",
-    "pattern.png",
-    "signlanguage.mp4",
-    "handwash.png",
-/* removed for covid-safe reasons */
-    // "cart.png",
-    // "celebritysanta.mp4",
-    // "gondola.png",
-    // "hanginthere.png",
-    // "kiss.png",
-    // "pyramid.png",
-    // "santaticket.png",
-    // "stuck.png",
-    // "twins.png",
+    "babybird.png",
```

Comment:
What does the mask prefix mean? I couldn't find the images in the repository, are they in this repo or somewhere else?

Reply:
We have two versions of these assets in the static file directory. Ones ending with _mask are edited to have masks. E.g. https://firebasestorage.googleapis.com/v0/b/santa-api.appspot.com/o/feed%2Fbabybird.png?alt=media
and https://firebasestorage.googleapis.com/v0/b/santa-api.appspot.com/o/feed%2Fbabybird_mask.png?alt=media